jetty-memcached/jetty-memcached-sessions/src/main/java/org/eclipse/jetty/memcached/session/MemcachedSessionDataMap.java

```diff
@@ -140,9 +142,13 @@ public void setHeartbeats(boolean heartbeats)
     @Override
     public void initialize(SessionContext context)
     {
+        if (isStarted())
+            throw new IllegalStateException("Context set after MemcachedSessionDataMap started");
+        
         try
         {
-            _builder.setTranscoder(new SessionDataTranscoder());
+            _context = context;
+            _builder.setTranscoder(new SessionDataTranscoder(_context.getContext().getClassLoader()));
```

Comment:
there is no such constructor. typo?

Reply:
ooops, I didn't realize that constructor was one you added in your jetty-10.0.x-distribution-session-tests branch ;) Fixed now.